app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -197,6 +197,8 @@ def disconnect(vim)
       ca_file.unlink
       self.ca_file = nil
     end
+  rescue => err
+    _log.warn("Failed to disconnect: #{err}")
```

Comment:
Do we need to still cleanup the ca file?  Also should we set vim to nil?

Reply:
> Do we need to still cleanup the ca file?

Yes, done

> Also should we set vim to nil?

`vim` is a local variable in `#vim_collector` so setting to `nil` isn't necessary, it will be reset the next time the thread is started automatically